### PR TITLE
[ENH] Begin removing confusing abbreviations

### PR DESCRIPTION
**Pull request by 36000**

### CODE REVIEW

**Comment by 36000** [comment]
@arokem  this is ready for review/merge, as well as the GPU one

File: AFQ/api/bundle_dict.py [hunk 1]
```diff
@@ -313,164 +311,172 @@ def baby_bd():
...
+                                    'mahal': {'distance_threshold': 4}},
+        'Left Cingulum': {'cross_midline': False,
+                          'include': [templates['CGC_roi1_L'],
+                                      templates['CGC_roi2_L'],
+                                      templates['CGC_roi3_L']],
+                          'exclude': [],
+                          'space': 'template',
+                          'prob_map': templates['CGC_L_prob_map'],
+                          'mahal': {'distance_threshold': 4}},
+        'Right Cingulum': {'cross_midline': False,
```
Comment (arokem) [inline AFQ/api/bundle_dict.py:331]: And same here: 

```suggestion
        'Right Cingulum Cingulate': {'cross_midline': False,
```

File: examples/howto_examples/plot_acoustic_radiations.py [hunk 2]
```diff
@@ -49,12 +49,12 @@
 ar_rois = afd.read_ar_templates()
 
 bundles = abd.BundleDict({
-    "AR_L": {
+    "Left Acoustic": {
```
Comment (arokem) [inline examples/howto_examples/plot_acoustic_radiations.py:52]: Do you want to do this to be consistent with "Left Optic Radiation"
```suggestion
    "Left Acoustic Radiation": {
```

File: examples/howto_examples/plot_acoustic_radiations.py [hunk 3]
```diff
@@ -49,12 +49,12 @@
...
         "start": ar_rois["AAL_Thal_L"],
         "end": ar_rois["AAL_TempSup_L"],
         "cross_midline": False,
     },
-    "AR_R": {
+    "Right Acoustic": {
```
Comment (arokem) [inline examples/howto_examples/plot_acoustic_radiations.py:57]: And same here:
```suggestion
    "Right Acoustic Radiation": {
```

File: examples/howto_examples/plot_acoustic_radiations.py [hunk 4]
```diff
@@ -108,7 +108,7 @@
 # be navigated, zoomed, rotated, etc.
 
 bundle_html = my_afq.export("indiv_bundles_figures")
-plotly.io.show(bundle_html["NDARAA948VFH"]["AR_L"])
+plotly.io.show(bundle_html["NDARAA948VFH"]["Left Acoustic"])
```
Comment (arokem) [inline examples/howto_examples/plot_acoustic_radiations.py:111]: And here
```suggestion
plotly.io.show(bundle_html["NDARAA948VFH"]["Left Acoustic Radiation"])
```

File: AFQ/api/bundle_dict.py [hunk 5]
```diff
@@ -90,186 +77,197 @@ def default18_bd():
...
+        'Right Anterior Thalamic': {
+            'cross_midline': False,
+            'include': [templates['ATR_roi1_R'],
+                        templates['ATR_roi2_R']],
+            'exclude': [],
+            'space': 'template',
+            'prob_map': templates['ATR_R_prob_map'],
+            'start': templates['ATR_R_start'],
+            'end': templates['ATR_R_end']},
+        'Left Cingulum': {
```
Comment (arokem) [inline AFQ/api/bundle_dict.py:89]: We are actually usually only looking at what is called the "Cingulum Cingulate", or also known as the "dorsal cingulum". In older versions we also had something called "cingulum hippocampus", which is the "ventral cingulum". We might want to revisit that one at some point, but it used to be a pretty hard-to-find and noisy bundle. At any rate, this is:
```suggestion
        'Left Cingulum Cingulate': {
```

Comment (arokem) [reply AFQ/api/bundle_dict.py:89]: See e.g., https://academic.oup.com/scan/article/11/5/746/1753353

File: AFQ/api/bundle_dict.py [hunk 6]
```diff
@@ -90,186 +77,197 @@ def default18_bd():
...
+            'cross_midline': False,
+            'include': [templates['CGC_roi1_L'],
+                        templates['CGC_roi2_L']],
+            'exclude': [],
+            'space': 'template',
+            'prob_map': templates['CGC_L_prob_map'],
+            'start': templates['CGC_L_start']},
+        'Right Cingulum': {
```
Comment (arokem) [inline AFQ/api/bundle_dict.py:97]:
```suggestion
        'Right Cingulum Cingulate': {
```

File: AFQ/api/bundle_dict.py [hunk 7]
```diff
@@ -313,164 +311,172 @@ def baby_bd():
...
+        'Right Anterior Thalamic': {'cross_midline': False,
+                                    'include': [
+                                        templates['ATR_roi1_R'],
+                                        templates['ATR_roi2_R'],
+                                        templates['ATR_roi3_R']],
+                                    'exclude': [],
+                                    'space': 'template',
+                                    'prob_map': templates['ATR_R_prob_map'],
+                                    'mahal': {'distance_threshold': 4}},
+        'Left Cingulum': {'cross_midline': False,
```
Comment (arokem) [inline AFQ/api/bundle_dict.py:323]: Same here:
```suggestion
        'Left Cingulum Cingulate': {'cross_midline': False,
```

**Review by arokem — COMMENTED** [review]
Overall, this makes the code a little bit more wordy, but I think can be less confusing, so overall an improvement. I had a couple of comments about some of the naming conventions used.

**Review by arokem — COMMENTED** [review]
Why are the CI failing? And why will this work once merged? Should we change the GHA to not run on PRs? Or will this work on subsequent PRs?

**Comment by 36000** [comment]
Assuming this is for the GPU PR. Its that the docker stuff is not supposed to run on branches, it pulls and builds master. So once that GPU PR is merged those should work. I am not sure why the GPU PR is still running those tests.

**Comment by arokem** [comment]
Yeah - sorry - I did mean to post that to #1069 and not here.

**Comment by arokem** [comment]
Whoops, I mean #1091, of course.

**Review by arokem — APPROVED** [review]

**Pull request merged**